src/surfaceComponent/Card/Card.tsx

```diff
@@ -0,0 +1,52 @@
+import { PropsWithChildren } from "react";
+import * as React from "react";
+import imageUrl from "../../static/images/card/card.png";
+import * as styles from "./Card.module.css";
+
+interface CardProps {
+  fullWidth?: boolean;
+}
+
+export const CardImage = () => {
+  return (
+    <div className={styles.imageWrapper}>
+      <img src={imageUrl} alt="placeholder" />
+    </div>
+  );
+};
+
+export const CardContent = (props: PropsWithChildren<never, never>) => {
```

Comment:
**nitpick**: `props: PropsWithChildren<{}>)` will be enough

Reply:
this one does not go through lint